app/views/layouts/application.html.slim

```diff
@@ -13,13 +13,13 @@ html lang='en'
 
   body
     = render 'layouts/header'
-    br
+    .br
```

Comment:
Rather wrap the notice, and yield in a .content-wrapper and add the spacing later